public/assets/images/icons/project/ManagedRegeneration.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 const ManagedRegeneration = () => {
   return (
-    <div style={{ display: 'flex', gap: 0 }}>
+    <div style={{ display: 'flex', gap: 0, marginBottom: '0.1em' }}>
```

Comment:
This is because defining a style in the common container for all icons :  `projectEcosystemOrTypeContainer` is not working for this icon specifically due to difference in it's positioning